### ISSUE TITLE
Wrong default user / group

### ISSUE THREAD

**Issue by Artanicus** [jitesoft/docker-lighttpd#1]
https://github.com/jitesoft/docker-lighttpd/blob/4b6c63b0604a41fd2b0a638fa87a56b8a2f9326f/conf.d/000-default.conf#L6

While the Docker file now defaults to www-data, the default conf.d values do not. This will result in a failure:
```
2021-07-27 11:41:44: server.c.1351) can't find groupname lighttpd
```

**Comment by Johannestegner** [comment]
Sorry for slow reply, github had pushed down this notification a whole lot so I totally missed it!
You are correct, will take a look at this right away. Tyvm!

**Comment by Johannestegner** [comment]
Fixed in 43ff32ca93d548add1b95eebc8d08d431ad6c272
Tyvm for the report!
